api/src/service/RpcClient.ts

```diff
@@ -117,7 +124,16 @@ export class RpcClient {
             durations.set(countKey, (durations.get(countKey) || 0) + elapsedMilliseconds);
             count.set(countKey, (count.get(countKey) || 0) + 1);
           }
-          resolve(resp.data.result);
+          let responseData = resp.data.result;
+          if (method === "liststreamitems" || method === "liststreamkeyitems") {
```

Comment:
To improve the code a little: you could check the method only once in an if-statement `if (method === "liststreamitems" || method === "liststreamkeyitems")`and check the `config.encryptionKey` inside